PyPDF2/_utils.py

```diff
@@ -66,23 +66,35 @@
 DEPR_MSG = "{} is deprecated and will be removed in PyPDF2 2.0.0. Use {} instead."
 
 
+def _isString(s):
```

Comment:
Decided to go with the camel case for these functions to make the diff a bit easier to read imo, and since these functions are not user facing and are going to be removed anyway in 2.0.0, it does not matter much on their naming convention imo.